integration-tests/funqy-google-cloud-functions/src/main/resources/application.properties

```diff
@@ -1,4 +1,2 @@
-quarkus.funqy.export=helloHttpWorld
-#quarkus.funqy.export=helloHttpWorldAsync
+quarkus.funqy.export=helloGCSWorld
```

Comment:
Is this change normal?

Reply:
This is a small cleanup.
The test has two functions in it, and 4 has been listed inside the application.properties.